src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -604,5 +637,6 @@
         padding: 0 !important;
         width: 0;
         flex: 0 0 auto !important;
+        min-width: 0 !important;
```

Comment:
Yes, we really need to get rid of anything that's not actual `width` to properly compute the width of a column. That includes paddings that have been defined by the user, for instance, so this is the only way we have to override them.